lib/bcdice/game_system/SwordWorld2_5.rb

```diff
@@ -66,9 +66,29 @@ class SwordWorld2_5 < SwordWorld2_0
 
         ・絡み効果表　(TT)
         　絡み効果表を出すことができます。
+
+        ・ドルイドの物理魔法用表　(Dru[2-6の値,7-9の値,10-12の値])
+        　例）Dru[0,3,6]+10-3
       INFO_MESSAGE_TEXT
 
-      register_prefix('H?K\d+.*', 'Gr(\d+)?', '2D6?@\d+.*', 'FT', 'TT')
+      register_prefix('H?K\d+.*', 'Gr(\d+)?', '2D6?@\d+.*', 'FT', 'TT', 'Dru\[\d+,\d+,\d+\].*')
+
+      def eval_game_system_specific_command(command)
+        case command
+        when /dru\[(\d+),(\d+),(\d+)\].*/i
+          power_list = (1..3).map { |i| Regexp.last_match(i).to_i }
```

Comment:
```suggestion
          power_list = Regexp.last_match.captures.map(&:to_i)
```

`MatchData#captures` を使うのが良いかと思います。
https://docs.ruby-lang.org/ja/latest/class/MatchData.html#I_CAPTURES